console/module/device/src/main/java/org/eclipse/kapua/app/console/module/device/client/device/tag/DeviceTabTags.java

```diff
@@ -41,6 +42,7 @@ protected void onRender(Element parent, int index) {
     @Override
     public void setEntity(GwtDevice selectedDevice) {
         super.setEntity(selectedDevice);
+        setEnabled(selectedDevice != null);
```

Comment:
The same should happen when deselecting a device; there's no need to disable this tab.